hotshot-query-service/migrations/postgres/V500__types_migration.sql

```diff
@@ -1,11 +1,13 @@
 CREATE TABLE leaf2
 (
-    height     BIGINT  PRIMARY KEY REFERENCES header (height) ON DELETE CASCADE,
+    height     BIGINT  REFERENCES header (height) ON DELETE CASCADE,
+    view       BIGINT,
```

Comment:
this should be `NOT NULL`, and also should have a index for it

Reply:
or maybe the PK index would take of it, I am not sure

Reply:
I don't think we lookup by view, so I think that means we don't need an index

Reply:
Yeah, I am just not sure if partial index performance < separate index, but okay lets leave it for now

Reply:
> I don't think we lookup by view, so I think that means we don't need an index

There is an `ORDER BY view` so that requires it.